kernel/idt.c

```diff
@@ -7,16 +7,15 @@
  * and interrupts
  *
  * created: 2022/10/18 - xlmod <glafond-@student.42.fr>
- * updated: 2022/10/21 - mrxx0 <chcoutur@student.42.fr>
+ * updated: 2022/11/25 - lfalkau <lfalkau@student.42.fr>
  */
 
 #include <stdint.h>
 #include <kernel/print.h>
 
 #include "idt_internal.h"
 
-__attribute__((aligned(0x10)))
-static t_idt_entry idt[256];
+static t_idt_entry *idt = (t_idt_entry *)0x0;
 static t_idt_ptr idtp;
 
```

Comment:
Using a define like `IDT_ADDRESS` can be preferable to future change of the address.

Reply:
Yep, same for the GDT